compiler/wasm/src/types/noir_artifact.ts

```diff
@@ -128,6 +128,16 @@ export interface DebugInfo {
   locations: Record<OpcodeLocation, SourceCodeLocation[]>;
 }
 
+/**
+ * The debug information for a given program.
+ */
+export interface ProgramDebugInfo {
```

Comment:
@Thunkar Requested your review just because I changed some things in the noir wasm compilation.